en/docs/design/api-policies/choreo-connect-policies/cc-op-policy-overview.md

```diff
@@ -0,0 +1,38 @@
+# Choreo Connect API Policies
+
+Choreo Connect supports API policies at the HTTP method level for request and response flows. Follow the steps given below to add policies to an API.
+
+1. Open the API Manager Publisher Portal.
+2. Select the API.
+3. Navigate to the **Policies** tab.
```

Comment:
Can we change step 3 to -> `Navigate to the  Policies page relevant to the API`
In global level also there's a Policies page/tab.